config/initializers/rack_attack.rb

```diff
@@ -1,124 +1,165 @@
-# Rails.logger.debug "Running initializer in #{__FILE__}"
+# frozen_string_literal: true
```

Comment:
Having this above the copyright will break our copyright update script.  Maybe we should revisit that at some point.  Does this still need to be on the first line of the file?

Reply:
Looks like this changed in ruby 3.1, now magic comments can be in the first 'comment section' rather than the first line. I tested this and it does seem to work.

I'll move the comment down under the copy

Reply:
Oh, awesome, thank you.